pubspec.lock

```diff
@@ -8,6 +8,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.0.10"
+  archive:
+    dependency: transitive
+    description:
+      name: archive
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "3.3.5"
+  args:
+    dependency: transitive
+    description:
+      name: args
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "2.3.1"
```

Comment:
Need to remove icons as we are only using standard flutter icons.

Reply:
Hi @kovarny ,

I have implemented review comments and pushed my changes. Request you to have a look.